android-components/components/service/contile/src/main/java/mozilla/components/service/contile/ContileTopSitesProvider.kt

```diff
@@ -169,9 +173,16 @@ class ContileTopSitesProvider(
         }
     }
 
+    private fun getCachedTopSitesIfValid(shouldUseServerMaxAge: Boolean = false) =
+        if (!isCacheExpired(shouldUseServerMaxAge = shouldUseServerMaxAge)) {
```

Comment:
nit: This is an example where I don't think it's necessary to use a named parameter, especially since the argument to the function already matches the parameter name. That is, I think it would be fine to write this as 
`if (!isCacheExpired(shouldUseServerMaxAge)`